### PR TITLE
docs(guide): fix broken anchor link

### PR DESCRIPTION
**Pull request by WLun001**
**Description:**
fix broken anchor link at guide (https://rxjs.dev/guide/operators#creation-operators)
at this section
```
See the list of all static creation operators here.
```
The `all the static creation operators here` anchor link not working due to have same id reference
**Related issue (if exists):**

### CODE REVIEW

**Comment by WLun001** [comment]
It seems like the markdown anchor link not working. Do we need to change to 
```md
### <a id="join-creation-operators"></a>Join Creation Operators ###
```
instead?

File: docs_app/content/guide/operators.md [hunk 1]
```diff
@@ -121,7 +121,7 @@ There are operators for different purposes, and they may be categorized as: crea
 
 For a complete overview, see the [references page](/api).
 
-### Creation Operators
+### <a id="creation-operators-list"></a>Creation Operators ###
```
Comment (JWO719) [inline docs_app/content/guide/operators.md]: Is the added `###` really needed? Seem to be invalid markdown to me?

Comment (WLun001) [reply docs_app/content/guide/operators.md]: those are not needed. I have removed them.

**Comment by JWO719** [comment]
Thanks for improving the docs by providing the anchor links

**Pull request merged**